scripts/install.sh

```diff
@@ -308,7 +308,22 @@ fi
 
 echo -e "\n\nFHIR Works is deploying. A fresh install will take ~20 mins\n\n"
 ## Deploy to stated region
-yarn run serverless-deploy --region $region --stage $stage ${issuerEndpoint:+"--issuerEndpoint $issuerEndpoint"} ${oAuth2ApiEndpoint:+"--oAuth2ApiEndpoint $oAuth2ApiEndpoint"} ${patientPickerEndpoint:+"--patientPickerEndpoint $patientPickerEndpoint"} || { echo >&2 "Failed to deploy serverless application."; exit 1; }
+# yarn run serverless-deploy --region $region --stage $stage ${issuerEndpoint:+"--issuerEndpoint $issuerEndpoint"} ${oAuth2ApiEndpoint:+"--oAuth2ApiEndpoint $oAuth2ApiEndpoint"} ${patientPickerEndpoint:+"--patientPickerEndpoint $patientPickerEndpoint"} || { echo >&2 "Failed to deploy serverless application."; exit 1; }
+
+params=()
+if [[ ! -z ${issuerEndpoint+x} ]]; then
+    params+=(--issuerEndpoint ${issuerEndpoint})
+fi
+
+if [[ ! -z ${oAuth2ApiEndpoint+x} ]]; then
+    params+=(--oAuth2ApiEndpoint ${oAuth2ApiEndpoint})
+fi
+
+if [[ ! -z ${patientPickerEndpoint+x} ]]; then
+    params+=(--patientPickerEndpoint ${patientPickerEndpoint})
+fi
+
+yarn run serverless-deploy --region $region --stage $stage ${params[@]} || { echo >&2 "Failed to deploy serverless application."; exit 1; }
```

Comment:
Did deleting the default empty values for these options not fix the problem this change is attempting to solve?

Reply:
I just tested and no, deleting the default empty values for these options not fix the problem.